v2/models/resourceoperation.go

```diff
@@ -0,0 +1,20 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package models
+
+// ResourceOperation defines an operation of which a device is capable
+// This object and its properties correspond to the ResourceOperation object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-metadata/2.x#/ResourceOperation
+type ResourceOperation struct {
+	Index          string
```

Comment:
Same question about need for Index?

Reply:
Index Removed. Agree to remove it because there is no specific usage.